docker/k8s/orchestrator/Dockerfile

```diff
@@ -1,38 +0,0 @@
-# Copyright 2019 The Vitess Authors.
```

Comment:
We don't need Orchestrator anymore with `vtorc`.

Reply:
last pushed to docker hub 2 years ago.